diff -urN a/Makefile.am b/Makefile.am
--- a/Makefile.am	2024-11-01 02:07:00.000000000 -0300
+++ b/Makefile.am	2025-01-22 18:52:37.730630829 -0300
@@ -12,6 +12,7 @@
 		massif \
 		dhat \
 		lackey \
+		libsig \
 		none
 
 EXP_TOOLS = \
diff -urN a/configure.ac b/configure.ac
--- a/configure.ac	2024-11-01 02:07:00.000000000 -0300
+++ b/configure.ac	2025-01-22 18:53:01.295265053 -0300
@@ -5683,6 +5683,8 @@
    dhat/tests/Makefile
    lackey/Makefile
    lackey/tests/Makefile
+   libsig/Makefile
+   libsig/tests/Makefile
    none/Makefile
    none/tests/Makefile
    none/tests/scripts/Makefile
